docs/src/man/guide.md

```diff
@@ -61,13 +61,15 @@ a single empty file at the moment, but we'll be adding to it later on.
 Add the following to your `make.jl` file
 
 ```julia
+push!(LOAD_PATH,"../src/")
```

Comment:
Looking back over this this morning I think it's probably best not put the `push!` line here, since further up the page we've explicitly stated that `Example` is a normal package, and rather add a [`!!! note` block](http://docs.julialang.org/en/latest/manual/documentation/#admonitions) after the code block that draws attention to needing to add `push!` or use `include` directly to load non-packages.